libr/bin/format/elf/elf.c

```diff
@@ -3728,10 +3728,7 @@ static void _add_ehdr_section (RBinFile *bf, ELFOBJ *eo) {
 	ptr->vaddr = eo->baddr;
 	ptr->size = ehdr_size;
 	ptr->vsize = ehdr_size;
-	ptr->add = false;
```

Comment:
uhm i think this is wrong because it will make relocs to fail

Reply:
How so? Look at the other lines below as well, I refactored the boolean assignment.